.circleci/push.bash.j2

```diff
@@ -5,27 +5,27 @@ export NEW_POINT_RELEASE=true
 
 IFS="," read -ra DOCKER_TAGS \<<< "<< parameters.extra_tags >>"
 
-echo "DOCKER_TAGS: ${DOCKER_TAGS[@]}"
+echo "DOCKER_TAGS: " "${DOCKER_TAGS[@]}"
 
-if [[ "<< parameters.nightly_build >>" =~ (True|true) ]]; then
+if [[ "<< parameters.nightly_build >>" ]]; then
```

Comment:
I'm not sure this is correct: _any_ value will pass this:

```bash
$ bash -xc '[[ "false" ]] && echo passed'
+ [[ -n false ]]
+ echo passed
passed
```

Reply:
Good catch! This should work instead:

```bash
if "<< parameters.nightly_build >>"; then
```

```bash
❯ if [[ "false" ]]; then echo yes; else echo no; fi
yes

❯ if "false"; then echo yes; else echo no; fi
no

❯ if "true"; then echo yes; else echo no; fi
yes
```

Fix incoming.

Reply:
This only works if it is `true` or `false` as those are commands that bash runs - `True` would fail with "command not found"

Reply:
Huh, TIL. I didn't think quoted strings would be interpreted as commands. I'll fix this later today.